src/components/editor/Bindings/index.tsx

```diff
@@ -109,7 +111,7 @@ function BindingsMultiEditor({
             : false;
     }, [draftSpecs, entityType, resourceConfig]);
 
-    useUpdateEffect(() => {
+    useEffect(() => {
```

Comment:
The changes in this PR works out the issues that led us to use `useUpdateEffect()` to set the server update required flag (ref: [PR 406](https://github.com/estuary/ui/pull/406)). When testing this PR, I recommend keeping a close eye on the server update required flags.